### ISSUE TITLE
Language server missing in vscode 0.1.0 release

### ISSUE THREAD

**Issue by roderik** [oxc-project/oxc#1911]
On startup after installing the latest update

```
[Error - 10:35:36 AM] oxc client: couldn't create connection to server.
Launching server using command /Users/roderik/.cursor/extensions/oxc.oxc-vscode-0.1.0-darwin-arm64/target/release/oxc_language_server failed. Error: spawn /Users/roderik/.cursor/extensions/oxc.oxc-vscode-0.1.0-darwin-arm64/target/release/oxc_language_server ENOENT
```

The file is not there indeed

![CleanShot 2024-01-06 at 10 38 25@2x](https://github.com/oxc-project/oxc/assets/16780/ecc2c8de-ab1f-4eeb-9f90-e1c0a5706b4e)

**Comment by IWANABETHATGUY** [comment]
https://github.com/oxc-project/oxc/commit/3a6d986b05b62f2d6941d5b67ceea0d5bf637c39, should be fixed, can you try again?

**Comment by Boshen** [comment]
Investigation is on the way, it's pretty difficult to test vscode :-(

**Comment by Boshen** [comment]
Should be fixed in v0.1.1, please excuse us for the inconvenience.

**Comment by roderik** [comment]
Works, thx!
